### PR TITLE
fix login issue

### PR DESCRIPTION
**Pull request by meena-rathi**
Fix Login issue

### CODE REVIEW

File: src/views/LoginPage.jsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ const LoginPage = () => {
     try {
       const response = await loginUser(email, password);
       dispatch(login(response));
+      console.log(response)
```
Comment (LiskaC) [inline src/views/LoginPage.jsx:30]: please remove console.log

File: src/components/protecteRoute.jsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import { Navigate } from "react-router-dom";
 import { useAuth } from "../context/AuthoContext";
+import { useSelector } from "react-redux";
 
 export const ProtecteRoute = ({children}) => {
-    const {auth} = useAuth();
-    if (!auth) {
+    //const {auth} = useAuth();
```
Comment (LiskaC) [inline src/components/protecteRoute.jsx:7]: please remove commented-out code. You'll need to remove the import too

**Review by LiskaC — COMMENTED** [review]
this PR will allow the user to access the assessment form.
**as discussed, a second PR is needed to fix the Navigation for logged in users**

**Review by LiskaC — APPROVED** [review]
Looks good to me! 

Final step is to do the same check for a logged in user in the NavBar
If there is a logged in user, the tab Assessment should be shown and login and register should be hidden
If there is no logged in uesr, the navbar should  be as it is now - ie. login and register visible and Assessment hidden.

This should be really small, like a 5 line change :)

**Comment by meena-rathi** [comment]
> this PR will allow the user to access the assessment form.
> **as discussed, a second PR is needed to fix the Navigation for logged in users**

Do I need To create a new branch for Navigation for logged in users?

**Comment by LiskaC** [comment]
> > this PR will allow the user to access the assessment form.
> > **as discussed, a second PR is needed to fix the Navigation for logged in users**
> 
> Do I need To create a new branch for Navigation for logged in users?

@meena-rathi I'd say:
- merge this PR here right now
- pull the changes in your main branch
- create a new branch
- make the navbar change
- PR it here, I can check suepr quick and then we can deploy the whole thing

**Pull request merged**